includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -14,6 +14,21 @@
 	const META_NAME_FEE = 'Stripe Fee';
 	const META_NAME_NET = 'Net Revenue From Stripe';
 
+	/**
+	 * Checks to see if request is invalid and that
+	 * they are worth retrying.
+	 *
+	 * @since 4.0.5
+	 * @param array $error
+	 */
+	public function is_retryable_error( $error ) {
+		return (
+			'invalid_request_error' === $error->type ||
```

Comment:
@roykho just a note that in most cases `invalid_request_error` is used for incorrect parameters in which case retrying with the same parameters will return the same error. However, we also use this for trying to charge sources that currently aren't in a chargeable state in which case retrying would make sense.